module/Application/view/error/index.phtml

```diff
@@ -1,62 +1,58 @@
-<h1><?php echo $this->translate('An error occurred') ?></h1>
-<h2><?php echo $this->message ?></h2>
+<h1>An error occurred</h1>
+<h2><?= $this->message ?></h2>
 
-<?php if (isset($this->display_exceptions) && $this->display_exceptions): ?>
+<?php if (isset($this->display_exceptions) && $this->display_exceptions) : ?>
 
-<?php if(isset($this->exception) && $this->exception instanceof Exception): ?>
+    <?php if (isset($this->exception) && $this->exception instanceof Exception) : ?>
 <hr/>
-<h2><?php echo $this->translate('Additional information') ?>:</h2>
-<h3><?php echo get_class($this->exception); ?></h3>
+<h2>Additional information:</h2>
```

Comment:
Actually, this is different.

The *markup* should not be indented here, only the *PHP statements*. Otherwise, when viewing the source of the page, you'll have markup indented that shouldn't be.

In this particular case, we're conditionally displaying some markup, but that markup is not semantically inside any other blocks.